frontend/biology4u/src/services/comment.service.js

```diff
@@ -27,6 +28,13 @@ class CommentService {
 
         return finalComments;
     }
+
+    addComment(comment) {
+        console.log(comment);
+        api.post('/comments', comment).then(() => {
+            router.go(0);

```

Comment:
Don't do that. It is better to return a promise, that way the consumer will know it the add comment succeded or failed and can do stuff according to the response.